config/locales/en.yml

```diff
@@ -95,7 +95,24 @@ en:
         page_time: 'Server time'
     home:
       index:
-        title: 'Welcome to Can I Go management system'
+        title: 'Can I Go?'
+        intro_html: 'In order to contribute to reducing the size of queues in stores and thus helping keep social distance, the application <i>Can I Go?</i> was born. See how you can contribute.'
+        store_owner: 'Own a shop?'
+        store_owner_1: 'Sign up to the website and get your store owner code.'
+        store_owner_2: 'Add or link to your store.'
```

Comment:
> link to your store

sounds like they will add an URL to their store. Maybe "Add or claim your store"? Not really sure of how this mechanism works to suggest a good verb.